memory/pmem_dm.py

```diff
@@ -202,8 +202,8 @@ def test(self):
 
     @avocado.fail_on(pmem.PMemException)
     def tearDown(self):
-        if hasattr(self, 'part'):
-            self.part.unmount()
+        if self.dm_disk:
+            process.system('umount %s' % self.dm_disk, ignore_status=True)
```

Comment:
@Pavithra1602 I think we can utility for unmounting the the disk.
can you please check this

Reply:
@Naresh-ibm Currently we are using the unmount utility in code but somehow it is not trying to unmount the pmem partition and says it is not mounted because of this namespace destroy is failing

 (2/2) /home/OpTest/avocado-fvt-wrapper/tests/avocado-misc-tests/memory/pmem_dm.py:PmemDeviceMapper.test;run-fio_job-fio_url-package-split-no_split-9c63: STARTED
 (2/2) /home/OpTest/avocado-fvt-wrapper/tests/avocado-misc-tests/memory/pmem_dm.py:PmemDeviceMapper.test;run-fio_job-fio_url-package-split-no_split-9c63:  ERROR: PMemException('Namespace destroy command failed') (0.32 s)

Reply:
@Pavithra1602 thanks Pavithra , let us merge this to clean run for CR and utility part we can fix later